src/Forms/GridField/GridFieldDetailForm_ItemRequest.php

```diff
@@ -591,10 +592,21 @@ public function Breadcrumbs($unlinked = false)
 
         if ($this->record && $this->record->ID) {
             $title = ($this->record->Title) ? $this->record->Title : "#{$this->record->ID}";
-            $items->push(new ArrayData(array(
+            $data = [
                 'Title' => $title,
-                'Link' => $this->Link()
-            )));
+                'Link' => $this->Link(),
+            ];
+            if ($this->record->hasExtension(Versioned::class) && $this->record->config()->get('versioned_gridfield_extensions')) {
```

Comment:
Can we put this into VersionedGridFieldItemRequest instead?

Reply:
Good idea